release-notes/release-6.17.0.md

```diff
@@ -0,0 +1,34 @@
+## Release summary
+
+Eclipse Che 6.17 includes:
+
+* **Workspace counts per status as metrics**: Allowing to monitor the workspace statuses with Prometheus-based tooling
+
+
+## Upgrading
+
+Instructions on how to upgrade.
+
+
+## Release details
+
+### Workspace counts per status as metrics
+
+The Prometheus metrics exposed by the Che server now include the `che_workspace_status` metric which
```

Comment:
When we have the 2 PRs under a single heading, we will want to avoid repetition.

"As part of the improvements in the monitoring and tracing of workspaces, we have added several new metrics in this release. The Prometheus metrics exposed by the Che server now include the `che_workspace_status` and `che_workspace_failure_total` metrics, allowing users to track the count of workspaces per status, and the total number of failed workspaces. Server operators can see when workspaces are failing by using a `while` tag, which recognizes `STARTING`, `RUNNING` or `STOPPING` values respectively. These new metrics enable operators to proactively identify when there are issues with workspaces."

Just a suggested alternative text.